src/arviz_plots/style.py

```diff
@@ -13,16 +13,79 @@ def use(name):
     -----
     There are some backends where default styles/templates are not supported.
     """
+    ok = False
+
     try:
         import matplotlib.pyplot as plt
 
-        plt.style.use(name)
+        if name in plt.style.available:
+            plt.style.use(name)
+            ok = True
     except ImportError:
         pass
 
     try:
         import plotly.io as pio
 
-        pio.templates.default = name
+        if name in pio.templates:
+            pio.templates.default = name
+            ok = True
+    except ImportError:
+        pass
+
+    if not ok:
+        raise ValueError(f"Style {name} not found.")
+
+
+def available():
+    """List available styles."""
+    try:
+        import matplotlib.pyplot as plt
+
+        print("Matplotlib styles:")
+        print(plt.style.available)
+        print()
     except ImportError:
         pass
+
+    try:
+        import plotly.io as pio
+
+        print("Plotly templates:")
+        print(list(pio.templates))
+    except ImportError:
+        pass
+
+
+def get(name, backend="matplotlib"):
+    """Get the style/template with the given name.
+
+    Parameters
+    ----------
+    name : str
+        Name of the style/template to get.
+    backend : str
+        Name of the backend to use. Options are 'matplotlib' (default) and 'plotly'.
+    """
+    if backend not in ["matplotlib", "plotly"]:
+        raise ValueError(f"Default styles/templates are not supported for Backend {backend}")
+
+    if backend == "matplotlib":
+        try:
```

Comment:
Given the backend is an input, I think we can skip the try except here. Otherwise not having matplotlib installed but using matplotlib here would result in the function not doing anything instead of raising an error